tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ng.html

```diff
@@ -104,7 +104,7 @@
       [seriesMetadataMap]="chartMetadataMap"
       [xScaleType]="xScaleType"
       [yScaleType]="yScaleType"
-      [customXFormatter]="xAxisType === XAxisType.RELATIVE ? relativeXFormatter : undefined"
+      [customXFormatter]="getCustomXFormatter()"
```

Comment:
I think your PR now contains part of mine too. Rebase and make sure your history is near on top of master's.

Reply:
Done.